portfolio/src/main/webapp/script.js

```diff
@@ -138,9 +138,21 @@ function initMap() {
       },
     ],    
   });
+
   // The marker, positioned at schoolLocation
   const marker = new google.maps.Marker({
     position: schoolLocation,
     map: map,
+    title: "The University of Central Oklahoma"
+  });
+
+  // Info window.
+  const infowindow = new google.maps.InfoWindow({
+      content: '<p id="describe">The University of Central Oklahoma is the Oklahoma\'s oldest institution of higher learning founded in 1890.</p>'
```

Comment:
Grammatically, I think it would read more clearly without the "the" before "Oklahoma's oldest".

Reply:
I will create a new PR implementing the suggested change.